plugins/MauticSocialBundle/Integration/SocialIntegration.php

```diff
@@ -26,11 +40,60 @@ abstract class SocialIntegration extends AbstractIntegration
     protected $integrationHelper;
 
     /**
-     * @param IntegrationHelper $integrationHelper
+     * @param EventDispatcherInterface $eventDispatcher
+     * @param CacheStorageHelper       $cacheStorageHelper
+     * @param EntityManager            $entityManager
+     * @param Session                  $session
+     * @param RequestStack             $requestStack
+     * @param Router                   $router
+     * @param DataCollectorTranslator  $translator
+     * @param Logger                   $logger
+     * @param EncryptionHelper         $encryptionHelper
+     * @param LeadModel                $leadModel
+     * @param CompanyModel             $companyModel
+     * @param PathsHelper              $pathsHelper
+     * @param NotificationModel        $notificationModel
+     * @param FieldModel               $fieldModel
+     * @param IntegrationEntityModel   $integrationEntityModel
+     * @param IntegrationHelper        $integrationHelper
      */
-    public function setIntegrationHelper(IntegrationHelper $integrationHelper)
-    {
+    public function __construct(
+        EventDispatcherInterface $eventDispatcher,
+        CacheStorageHelper $cacheStorageHelper,
+        EntityManager $entityManager,
+        Session $session,
+        RequestStack $requestStack,
+        Router $router,
+        DataCollectorTranslator $translator,
```

Comment:
@hluchas based on the message @anton-vlasenko posted this param is not correct type. Shouldn't it be this isntead?
```suggestion
        TranslatorInterface $translator,
```

Reply:
That's it. I've the same problem in other class. TY!

Reply:
@hluchas It has a wrong type and it could be seen from the source code. I agree with @escopecz. You don't even need test steps to fix it.
I will check again.